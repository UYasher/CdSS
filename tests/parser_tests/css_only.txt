b {
    height: 1px;
}