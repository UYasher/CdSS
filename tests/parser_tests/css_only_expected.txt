start
  instruction
    css b
  instruction
    css {
  instruction
    css h
  instruction
    css e
  instruction
    css i
  instruction
    css g
  instruction
    css h
  instruction
    css t
  instruction
    css :
  instruction
    css 1
  instruction
    css p
  instruction
    css x
  instruction
    css ;
  instruction
    css }